packages/fxa-settings/src/models/Account.ts

```diff
@@ -1364,5 +1364,9 @@ export class Account implements AccountData {
     return data;
   }
 
-  async removeRecoveryPhone() {}
+  async removeRecoveryPhone() {
+    return await this.withLoadingStatus(
+      this.authClient.recoveryPhoneDelete(sessionToken()!)
+    );
```

Comment:
Missing cache update that was pushed to the demo branch? or alternatively a refresh of the recoveryPhone state?

Reply:
Preferably we just update apollo cache because we know the state, to avoid that extra network call.